masterlist.yaml

```
@@ -18836,6 +18942,24 @@ plugins:
   - name: 'JKs_The_Winking_Skeever_AlternateStartMods.esp'
     after: [ 'Alternate Start - Live Another Life.esp' ]
 
+  - name: 'JK''s Understone Keep.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/55571/'
+        name: 'JK''s Understone Keep'
+    msg: [ *compatPatch_JKIPC ]
+
+  - name: 'JK''s White Phial.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/47713/'
+        name: 'JK''s White Phial'
+    msg: [ *compatPatch_JKIPC ]
+
+  - name: 'JK''s Warmaiden''s.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/33685/'
+        name: 'JK''s Warmaiden''s'
+    msg: [ *compatPatch_JKIPC ]
```

These two should be sorted alphabetically.

```suggestion
  - name: 'JK''s Warmaiden''s.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/33685/'
        name: 'JK''s Warmaiden''s'
    msg: [ *compatPatch_JKIPC ]

  - name: 'JK''s White Phial.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/47713/'
        name: 'JK''s White Phial'
    msg: [ *compatPatch_JKIPC ]
```